src/core/View.js

```diff
@@ -386,6 +392,11 @@ export default class View extends Component {
 
   componentDidUpdate(prevProps, prevState, snapshot) {
     this.update(this.props, prevProps);
+
+    // Cube Axes
+    if (this.props.showCubeAxes && this.cubeAxes == null) {
+      this.initCubeAxes();
+    }
```

Comment:
This gets called after `this.update`, which means if `this.props.showCubeAxes` is set to true, [line 482](https://github.com/Kitware/react-vtk-js/pull/70/files#diff-b7267f7f1a27837a979426c002ada36d97925bce54b0388d65127314c21ff342R482) will try to deref `this.cubeAxes`, which will be null. This check should happen before the `if (this.props.showCubeAxes)` logic.

Reply:
Could you please update the PR yourself? I'm on PTO this week